src/streamed/index.ts

```diff
@@ -176,6 +176,7 @@ export class StreamedOrderbook {
    * Apply new confirmed events to the account state and store the remaining
    * events that are subject to reorgs into the `pendingEvents` array.
    *
+   * @param toBlock optional block number until which to fetch events
```

Comment:
ubernit: tsdoc preferes a `-` between the parameter name and description
```suggestion
   * @param toBlock - Optional block number until which to fetch events
```